core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -1291,7 +1311,8 @@ private[ui] class TaskPagedTable(
         } else {
           Nil
         }} ++
-        Seq(("Errors", ""))
+        Seq(("Errors", "")) ++
+        Seq(("Logs", ""))
```

Comment:
Should we possibly call this Executor Logs instead? It could clarify that the log isn't just for this stage but the whole log for the executor it was run on.